src/Discovery/DiscoveryConfig.d.ts

```diff
@@ -15,6 +15,9 @@ export interface DiscoveryConfig {
             enableDownloadStudyMetadata?: boolean
             variableMetadataFieldName?: string
             enableDownloadVariableMetadata?: boolean
+            enableFillRequestForm?: boolean
+            fillRequestFormDisplayText?: string
+            fillRequestFormURL?: string
```

Comment:
missing `externalWebsiteName` and `externalWebsiteURL` in here?

Reply:
added